src/modules/dashboard/includes/Term_Entity_Match/Term_Query.php

```diff
@@ -57,12 +57,20 @@ public function __construct( $request, $cursor, $cursor_sort, $limit ) {
 				e.about_jsonld as match_jsonld,
 				t.name,
 				t.name as term_name,
-				e.id AS match_id
+				e.id AS match_id,
+				e.match_name,
+		        (
+					SELECT COUNT(*)
+					FROM {$wpdb->prefix}term_relationships tr
+					INNER JOIN {$wpdb->prefix}posts p ON tr.object_id = p.ID
+					WHERE tr.term_taxonomy_id = tt.term_taxonomy_id
+					AND p.post_status = 'publish'
```

Comment:
This query would be too heavy and slow on websites where we have thousands of terms/posts.